examples/inputs/service_chain_1-example-inputs.yaml

```diff
@@ -0,0 +1,40 @@
+# proxy plugin configuration
```

Comment:
is it OK to make all of these defaults to the inputs instead of requiring an inputs file?